### PR TITLE
#10 add new form

### LINKED ISSUE #10: Able to add new fish species
Create a form where we can add new fish species. Be sure to include fields related to the threat and details of the fish species. This page is linked in the top menu as `/new`. Bear in mind this relates to the server side work of #12

### PR DESCRIPTION
**Pull request by linches**
Added form component to add new fish.
Form is working on the client.

closes #10

### CODE REVIEW

File: client/components/AddNew.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  handleChange = (e, { value }) => this.setState({ value })
+
+  render() {
+    const { value } = this.state
+    return (
+      <Form>
+        <Form.Group widths='equal'>
+          <Form.Input fluid label='Name' placeholder='Name' />
+          <Form.Input fluid label='Threat' placeholder='Threat' />
```
Comment (richchurcher) [inline client/components/AddNew.jsx]: Nice use of Semantic UI!

Comment (richchurcher) [reply client/components/AddNew.jsx]: However, these aren't hooked up to the state yet. To do that, we'd use something like:
```
<Form.Input fluid label='Threat' placeholder='Threat' onChange={this.handleChange} value={this.state.threat} />
```

File: client/components/AddNew.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import { Form } from 'semantic-ui-react'
+import { Link } from 'react-router-dom'
+
+export default class AddNew extends Component {
+  state = {}
+
+  handleChange = (e, { value }) => this.setState({ value })
```
Comment (richchurcher) [inline client/components/AddNew.jsx]: This is a good start, you're on the right track. Remember computed property syntax?
```js
handleChange = e => this.setState({ [ e.target.name ]: e.target.value })
```

File: client/components/AddNew.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import { Form } from 'semantic-ui-react'
+import { Link } from 'react-router-dom'
+
+export default class AddNew extends Component {
+  state = {}
```
Comment (richchurcher) [inline client/components/AddNew.jsx]: Don't forget to include the default form values in here, like:
```js
state = {
  username: '',
  threat: ''
}
```
or whatever.

Comment (linches) [reply client/components/AddNew.jsx]: Added username and threat to state.

File: client/components/AddNew.jsx (hunk 2 above)
Comment (linches) [reply client/components/AddNew.jsx]: Added changes to handleChange event

File: client/components/AddNew.jsx (hunk 1 above)
Comment (linches) [reply client/components/AddNew.jsx]: Hooked up to the state with handlechange.

**Pull request merged**